### PR TITLE
Tried to simplify the whole setup of this.

### PR DESCRIPTION
**Pull request by treeder**
Removes ngrok and various other steps. 

This is pretty mac specific as it uses the `docker.for.mac.localhost` host. To support linux, should use `--add-host` or something: https://stackoverflow.com/questions/28056522/access-host-database-from-a-docker-container/28058549

### CODE REVIEW

File: scripts/setenv.sh [hunk 1]
```diff
@@ -1,10 +1,10 @@
 fn apps config set myapp PUBNUB_PUBLISH_KEY $PUBNUB_PUBLISH_KEY
 fn apps config set myapp PUBNUB_SUBSCRIBE_KEY $PUBNUB_SUBSCRIBE_KEY
-fn apps config set myapp FUNC_SERVER_URL ${API_URL}/r/myapp
+fn apps config set myapp FUNC_SERVER_URL ${FUNC_SERVER_URL}/r/myapp
 fn apps config set myapp MINIO_SERVER_URL $MINIO_SERVER_URL
+fn apps config set myapp ACCESS $MINIO_ACCESS_KEY
```
Comment (carimura) [inline scripts/setenv.sh]: it's actually not technically a Minio-specific keypair... the whole thing works with S3 as well with no code changes hence the names ACCESS/SECRET

**Comment by treeder** [comment]
Yes, or anything else running on the host.

File: scripts/setenv.sh (hunk 1 above)
Comment (treeder) [reply scripts/setenv.sh]: Oh, how about STORAGE_ACCESS_KEY or something then? access and secret aren't clear what it's for.

File: services/draw/Dockerfile [hunk 2]
```diff
@@ -2,8 +2,10 @@ FROM ruby:2.4.0-alpine
 
 RUN apk add --update alpine-sdk ca-certificates && rm -rf /var/cache/apk/*
 RUN apk add --update imagemagick-dev imagemagick && rm -rf /var/cache/apk/*
+
 WORKDIR /func
-ADD . /func/
+ADD Gemfile* /func/
```
Comment (treeder) [inline services/draw/Dockerfile]: This speeds up builds by caching the bundle install.

File: scripts/setenv.sh (hunk 1 above)
Comment (carimura) [reply scripts/setenv.sh]: ya that works. did you search the code for references to that ENV var? mind making those changes?

**Comment by carimura** [comment]
I also noticed you switched the minio setup step to go in front of vista setup.. problem is /publish has to be deployed *before* minio setup because minio won't start unless it can hit it's webhook endpoints. Can you reverse order back? (or provide other suggestion)

**Comment by treeder** [comment]
Doesn't seem to be an issue for me, even when fn server isn't running, the webhook still gets created.. ?

**Pull request merged**